File: src/main/java/calebzhou/rdi/core/client/model/GeoLocation.java
package calebzhou.rdi.core.client.model;

import calebzhou.rdi.core.client.util.RdiSerializer;

import java.io.Serializable;

/**
 * Created by calebzhou on 2022-09-20,16:16.
 */
public class GeoLocation implements Serializable {
	public double latitude;
	public double longitude;

	public GeoLocation(double latitude, double longitude) {
		this.latitude = latitude;
		this.longitude = longitude;
	}

	@Override
	public String toString() {
		return RdiSerializer.GSON.toJson(this);
	}
}

File: src/main/java/calebzhou/rdi/core/client/model/GeoLocation.kt
package calebzhou.rdi.core.client.model

/**
 * Created by calebzhou on 2022-10-03,21:09.
 */

/**
 *  地理天气
 */
//经度纬度
data class GeoLocation(val latitude:Double,val longitude:Double)
//RDI地址位置
data class RdiGeoLocation(val nation: String, val province: String, val city: String,
                     val district: String, val isp: String, val location: GeoLocation
)
